### PR TITLE
Pool-1116 Pool index pages not linked to the users wallet

### PR DESCRIPTION
**Pull request by ddesrosier**
- `/pools/[network]` will load the index page, for `network` regardless of what the users wallet is connected to
- Move the message about not being able to compile the full list of pools to a tooltip so it's less aggressive

### CODE REVIEW

File: lib/components/ChangeWalletNetworkNotificationBanner.jsx [hunk 1]
```diff
@@ -7,15 +7,14 @@ import { WalletContext } from 'lib/components/WalletContextProvider'
...
-import { ButtonLink } from 'lib/components/ButtonLink'
 
 export const ChangeWalletNetworkNotificationBanner = (props) => {
   const { walletConnected, walletMatchesNetwork, walletNetwork, view, chainId } = useNetwork()
 
   if (!walletConnected || walletMatchesNetwork) return null
 
   return (
-    <NotificationBanner className='bg-teal'>
+    <NotificationBanner className='bg-teal' canClose>
```
Comment (chuckbergeron) [inline lib/components/ChangeWalletNetworkNotificationBanner.jsx:17]: Why do we want them to be able to close this? Currently this is the only thing that let's them know that they're on the wrong network.

File: lib/components/ChangeWalletNetworkNotificationBanner.jsx [hunk 2]
```diff
@@ -49,7 +48,7 @@ const ChangeWalletNetworkNotification = (props) => {
     <div className='flex flex-col sm:flex-row justify-between items-center'>
       <span>
         👋 Your wallet is currently set to <b>{walletChainName}</b>. Please connect to{' '}
-        <b>{poolChainName}</b> to participate in this pool.
+        <b>{poolChainName}</b> to participate.
```
Comment (chuckbergeron) [inline lib/components/ChangeWalletNetworkNotificationBanner.jsx:51]: I liked the qualifier "in this pool" - if we leave it out, it sounds like you have to switch to that network to participate at all. (ie. please switch to Matic to participate)

File: lib/components/CustomYieldNotificationBanner.jsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const CustomYieldNotificationBanner = () => {
   if (prizePoolType !== PRIZE_POOL_TYPE.yield) return null
 
   return (
-    <NotificationBanner className='bg-purple-2'>
+    <NotificationBanner className='bg-purple-2' canClose>
```
Comment (chuckbergeron) [inline lib/components/CustomYieldNotificationBanner.jsx:13]: Another one I'd rather not let them be able to dismiss. This is important and people don't read, they just click X's.

I think it makes sense for users who are very competent: who would read that it's an unaudited yield source, know what that means and decide they want to stay and risk it, then close the warning. But very competent users are a bad gauge of what most need. 😉

**Review by chuckbergeron — CHANGES_REQUESTED** [review]

File: lib/components/ChangeWalletNetworkNotificationBanner.jsx (hunk 2 above)
Comment (ddesrosier) [reply lib/components/ChangeWalletNetworkNotificationBanner.jsx:51]: This notification shows up on the index page if you go to something like `/pools/matic` so having "in this pool" didn't quite make sense. And making this component award of context seemed more difficult than it was worth it.

File: lib/components/ChangeWalletNetworkNotificationBanner.jsx (hunk 1 above)
Comment (ddesrosier) [reply lib/components/ChangeWalletNetworkNotificationBanner.jsx:17]: - It takes up a decent amount of space so for users will small screens / mobile
- If I know my wallet can't connect to the network it's suggesting, it's just annoying

File: lib/components/CustomYieldNotificationBanner.jsx (hunk 3 above)
Comment (ddesrosier) [reply lib/components/CustomYieldNotificationBanner.jsx:13]: More of the same reasons as the other one. But I think it makes sense to force the tradeoff of space for the warning in this case.

**Pull request merged**